packages/vue/src/components/authenticator.vue

```diff
@@ -4,6 +4,7 @@
     data-amplify-authenticator
     v-if="!state?.matches('authenticated')"
   >
+    <slot name="header"> </slot>
```

Comment:
Will this ` ` space cause it to render anything by default?  I know that sometimes `padding` can get a "ghost element" when there's excess whitespace.

Reply:
OK, I just double checked, and it did NOT render anything there with the whitespace. So we should be fine.